recipes/zziplib/recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: zziplib
+  version: 0.13.69
+
+source:
+  url: https://github.com/gdraheim/zziplib/archive/v0.13.69.tar.gz
```

Comment:
Can you put the version as a Jinja variable in here? Could be that the autotick-bot needs it.

Reply:
done